concent_api/core/message_handlers.py

```diff
@@ -908,18 +908,21 @@ def store_subtask(
     return subtask
 
 
-def handle_messages_from_database(
-    client_public_key: bytes,
-    response_type: PendingResponse.Queue,
-) -> Union[message.Message, None]:
-    assert client_public_key    not in ['', None]
+def handle_messages_from_database(client_public_key: bytes) -> Union[message.Message, None]:
+    assert client_public_key not in ['', None]
 
     encoded_client_public_key = b64encode(client_public_key)
-    pending_response = PendingResponse.objects.select_for_update().filter(
-        client__public_key = encoded_client_public_key,
-        queue              = response_type.name,
-        delivered          = False,
+    pending_response = PendingResponse.objects.filter(
```

Comment:
are you sure we should remove `select_for_update` in this place? What is the reason?

Reply:
No reason, I did it by accident on rebase.